{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -42,19 +43,16 @@ repository = "https://github.com/Nieuwe-Warmte-Nu/{{cookiecutter.repo_name}}"
 changelog = "https://github.com/Nieuwe-Warmte-Nu/{{cookiecutter.repo_name}}/blob/main/CHANGELOG.md"
 
 [project.scripts]
-newchess = "{{cookiecutter.python_import_name}}.infrastructure.app:cmdline_app"
+{{cookiecutter.python_import_name}} = "{{cookiecutter.python_import_name}}.infrastructure.app:cmdline_app"
 
 [build-system]
 build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools ~= 67.7.2",
+    "setuptools ~= 69.0.3",
     "wheel ~= 0.40.0",
     "setuptools-git-versioning<2",
 ]
 
-[tool.setuptools]
```

Comment:
`tool.setuptools.packages`  needs a package list such as `{{python_import_name}}.main` instead of a file path. Caused issues with parsing pyproject.toml in various tools.

Reply:
But I think the packages directive is necessary to have anything packaged at all? 

Should be: `packages = ["{{python_import_name}}"]`  based on https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#

Reply:
Not necessary, happens automatically.